pages/windows/clip.md

```diff
@@ -5,8 +5,16 @@
 
 - Pipe command line output to the Windows clipboard:
 
-`{{dir}} | clip`
+`dir|clip`
 
 - Copy the contents of a file to the Windows clipboard:
 
-`clip < {{path/to/file}}`
+`clip<path/to/file.ext`
```

Comment:
Again, here the lack of spacing makes it harder to revert, and see also my comment about the token syntax `{{` `}}`.